### DISCUSSION TITLE
Index in the background / Search without indexing

### DISCUSSION THREAD

**Discussion by boredPanda96** [glowingjade/obsidian-smart-composer#185] [Ideas & Feature Requests]
It would be beneficial to have an option for indexing in the background. Currently, every time I open the Smart Composer, I have to wait for the indexing to complete. While it's important to search the most recent information, there are times when I need faster access and I know that the recent additions don't include the data I'm searching for. It would be helpful to have an option to search without rebuilding the index. 

Also thank you for developing this plugin!

**Comment by glowingjade** [comment]
Thanks for the suggestion. Will consider this. 👍

**Reply by boredPanda96** [reply]
Hi @glowingjade, is this feature on the roadmap? Would it be helpful if I opened an issue for it? It would be a valuable addition, as having to reindex before every new search—even after a minor change—can be quite inconvenient.

**Reply by glowingjade** [reply]
Yes, it would be helpful for you to open an issue. Thanks!
